system/core/document.php

```diff
@@ -182,6 +182,19 @@ public function writeScript($js) {
 			$script->appendChild($this->createTextNode(preg_replace('#<script[^>]*>(.*)</script>#isU','$1', $js)));
 		}
 
+		/*
+			Function: writeScript
+			adds content to the script element in the head section of the document, creates script element when needed
+
+			Parameters:
+			$js - javascript code to be added
+		*/
+		public function dump($var) {
+			$output = 'console.log.apply(console, ' . json_encode( $var ) . ');';
```

Comment:
This ensures a nicer log as it would with a regular console.log

Reply:
This would only work if `json_encode( $var )` always results in a JS array. Is that expected behavior? 
Note that using apply on internal functions (such as the console functions) is prohibited in some browsers. (ie IE).

Reply:
Was albe to fix this with a much simpler solution. :-)

Reply:
Ah, I had not realized that you're using apply for currying the array into function arguments. And indeed, the new solution seems cleaner, thanks :-)